grains_of_sand/grains_of_sand.py

```diff
@@ -0,0 +1,14 @@
+import sys
+
+num_cases = int(sys.stdin.readline().rstrip())
+
+for _ in range(num_cases):
+    num_teams = int(sys.stdin.readline().rstrip())
+
+    total = 0
+
+    for _ in range(num_teams):
+        count = int(sys.stdin.readline().rstrip())
```

Comment:
This is great! One small optimization though if count is not going to be used any other way you may just want to use total.

total += int(sys.stdin.readline().rstrip())

collapses the logic a bit but honestly both are perfectly fine